.travis.yml

```diff
@@ -25,14 +25,15 @@ matrix:
       script: |
         cargo install clippy
         cargo clippy --all -- -D clippy-pedantic
+        cd coresimd
+        cargo clippy --all -- -D clippy-pedantic
```

Comment:
cargo clippy --all` only works for the crates in a workspace, but `coresimd` and `stdsimd` are not in the same workspace yet: https://github.com/rust-lang-nursery/rust-clippy/issues/2238